<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
  Copyright (c) the JPEG XL Project Authors. All rights reserved.

  Use of this source code is governed by a BSD-style
  license that can be found in the LICENSE file.
 </comment>
 <class>CodecComparisonWindow</class>
 <widget class="QMainWindow" name="CodecComparisonWindow">
  <property name="windowTitle">
   <string>Codec Comparison Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0,1">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="1">
         <widget class="QComboBox" name="imageSetComboBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="imageSetLabel">
          <property name="text">
           <string>Image set:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="imageLabel">
          <property name="text">
           <string>Image:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="imageComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QComboBox" name="compressionLevel1ComboBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="codec1ComboBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="match1Label">
          <property name="text">
           <string>&lt;a href=&quot;#match1&quot;&gt;Match →&lt;/a&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="match2Label">
          <property name="text">
           <string>&lt;a href=&quot;#match2&quot;&gt;Match ←&lt;/a&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="compressionLevel2ComboBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="codec2ComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1,0">
      <item>
       <widget class="QLabel" name="size1Label">
        <property name="text">
         <string>No image loaded.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="renderingModeLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="size2Label">
        <property name="text">
         <string>No image loaded.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="jpegxl::tools::SplitImageView" name="splitImageView" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>jpegxl::tools::SplitImageView</class>
   <extends>QWidget</extends>
   <header>split_image_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
